litr/src/main/java/com/linkedin/android/litr/frameextract/FrameExtractJob.kt

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2021 LinkedIn Corporation
+ * All Rights Reserved.
+ *
+ * Licensed under the BSD 2-Clause License (the "License").  See License in the project root for
+ * license information.
+ */
+package com.linkedin.android.litr.frameextract
+
+import android.graphics.Bitmap
+import android.media.ThumbnailUtils
+import android.util.Log
+import com.linkedin.android.litr.ExperimentalFrameExtractorApi
+import com.linkedin.android.litr.frameextract.behaviors.FrameExtractBehavior
+import com.linkedin.android.litr.frameextract.behaviors.FrameExtractBehaviorFrameListener
+
+/**
+ * Provides the request lifecycle for extracting video frames. The specifics of extraction work are delegated to [FrameExtractBehavior]s.
+ */
+@ExperimentalFrameExtractorApi
```

Comment:
This is exposed externally? We should make these `internal`
Note to self: In general, we should change our build config to enforce explicit access qualifiers in Kotlin classes, to prevent clients from using stuff that is not meant to be exposed.

Reply:
Ah good catch, done. I also made the executor/task related classes internal (they were originally being injected so they were open)